### ISSUE TITLE
Documentation: Misleading documentation for LokiExporter README and Example

### ISSUE THREAD

**Issue by MacNale** [open-telemetry/opentelemetry-collector-contrib#2943]
**Describe the bug**
README of the lokiexporter suggests that, the exporter supports "logs" pipline. However, the example (https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/testdata/config.yaml) talks about "metrics" pipeline.
```service:
  pipelines:
    metrics:
      receivers: [ nop ]
      processors: [ nop ]
      exporters: [ loki, loki/allsettings ]
```

**What did you expect to see?**
A clear and concise description how to export logs to loki using loki exporter.

**What did you see instead?**
Dont know how to use loki exporter. When to use loki exporter?

**Comment by gramidt** [comment]
Thank you for finding this, @MacNale! I'll update this now.

For a better example, I would look at the exporter/lokiexporter/example folder.

**Comment by gramidt** [comment]
@bogdandrutu @tigrannajaryan - Please close this issues as it was resolved with #2997. I forgot to put "fixes #2943" in the description.
